ui/js/filters.js

```diff
@@ -183,3 +183,9 @@ treeherder.filter('displayPrecision', function() {
         return parseFloat(input).toFixed(2);
     };
 });
+
+treeherder.filter('absoluteValue', function() {
```

Comment:
What a pain to have to create a filter just for this. :/

(I can't find a better way though, so this is fine)